MapboxNavigation/NavigationMapView.swift

```diff
@@ -161,10 +161,16 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         let durationUntilNextManeuver = stepProgress.durationRemaining
         let durationSincePreviousManeuver = expectedTravelTime - durationUntilNextManeuver
         
-        if !isPluggedIn,
-            durationUntilNextManeuver > FrameIntervalOptions.durationUntilNextManeuver,
+        guard !isPluggedIn else {
+            frameInterval = FrameIntervalOptions.pluggedInFrameInterval
+            return
+        }
+    
+        if durationUntilNextManeuver > FrameIntervalOptions.durationUntilNextManeuver,
```

Comment:
Let’s use `&&` here instead of `,`. In the absence of a `let` expression, the comma tends to obscure the fact that both cases must evaluate to `true`.